bhagavad_gita_api/data/insert/translations.py

```diff
@@ -19,12 +19,12 @@
 for i in track(data, description="Loading translations"):
     li.append(
         GitaTranslation(
-            description=i.get("description"),
-            author_name=i.get("authorName"),
-            language=i.get("lang"),
-            verse_id=i.get("verseNumber"),
-            author_id=i.get("author_id"),
-            language_id=i.get("language_id"),
+            description=i["description"],
+            author_name=i["authorName"],
+            language=i["lang"],
+            verse_id=i["verseNumber"],
```

Comment:
actually verse_number is comming from scraped data and verse_id is foriegn key thing 
now we do need both of these here because the scripts used to make references used verse_number to setup verse_id , 
now we don't have referencing scripts but this thing remains essential to reconstruct database if anything breaks in future

Reply:
Hmm, `verse_number` is a blank column though.

Reply:
Maybe you can populate `verse_number` instead and make that a foreign key.

Reply:
Sure we'll do that and sorry about the changing of order , i think it got messed up while commiting changes

Reply:
Thanks a lot!